build/certificates/README.md

```diff
@@ -0,0 +1,105 @@
+# IBM FHIR Server - Development Certificates 
+
+The certificates are used in development and CI/CD.
+These certificates are not configured for production use.
+
+The sections in this document walks through creation of the certificates in a temporary folder, and then subsequent update inline.  
+
+The Audit Kafka jks are updated separately. 
+
+# Create CA and Certs and P12s (does not replace)
+
+This script should be run first, and subsequently each many section must update their cert/key. 
```

Comment:
Not sure what "each many section" means.

Reply:
fixed